tests/ragger/test_sign.py

```diff
@@ -1,67 +1,372 @@
+                         backend: BackendInterface,
+                         navigator: Navigator,
+                         test_name: str):
+    tx_params: dict = {
+        "nonce": 1,
+        "gasPrice": Web3.to_wei(GAS_PRICE2, 'gwei'),
+        "gas": GAS_LIMIT,
+        "to": ADDR2,
+        "value": 31415926913374232,
+        "chainId": 56
+    }
+    common(firmware, backend, navigator, tx_params, test_name, BIP32_PATH2)
+
+
+# Legacy Zemu ChainId: Transfer on network 112233445566 on Ethereum
+def test_legacy_chainid(firmware: Firmware,
+                         backend: BackendInterface,
+                         navigator: Navigator,
+                         test_name: str):
+    tx_params: dict = {
+        "nonce": NONCE2,
+        "gasPrice": Web3.to_wei(GAS_PRICE, 'gwei'),
+        "gas": GAS_LIMIT,
+        "to": ADDR2,
+        "value": 31415926913374232,
+        "chainId": 112233445566
+    }
+    common(firmware, backend, navigator, tx_params, test_name, BIP32_PATH2)
+
+
+# Legacy Zemu contract data warning: Try to blind sign with setting disabled
+def test_legacy_contract(firmware: Firmware,
+                         backend: BackendInterface,
+                         navigator: Navigator,
+                         test_name: str):
+
+    buffer = bytes.fromhex("058000002c8000003c800000010000000000000000f849208506fc23ac008303dc3194f650c3d88d12db855b8bf7d11be6c55a4e07dcc980a4a1712d6800000000000000000000000000000000000000000000000000000000000acbc7018080")
+    app_client = EthAppClient(backend)
+
+    try:
+        with app_client.send_raw(0xe0, 0x04, 0x00, 0x00, buffer):
+            if firmware.device.startswith("nano"):
+                # No screens recorded on Stax
+                moves = [
+                    NavInsID.RIGHT_CLICK,
+                    NavInsID.BOTH_CLICK
+                ]
+                navigator.navigate_and_compare(ROOT_SNAPSHOT_PATH,
+                                                test_name,
+                                                moves)
+
+    except ExceptionRAPDU as e:
+        assert e.status == StatusWord.INVALID_DATA
 
 
 def test_1559(firmware: Firmware, backend: BackendInterface, navigator: Navigator):
-    common(firmware, backend, navigator, {
+    tx_params: dict = {
         "nonce": NONCE,
         "maxFeePerGas": Web3.to_wei(145, "gwei"),
         "maxPriorityFeePerGas": Web3.to_wei(1.5, "gwei"),
         "gas": GAS_LIMIT,
         "to": ADDR,
         "value": Web3.to_wei(AMOUNT, "ether"),
         "chainId": CHAIN_ID
-    })
+    }
+    common(firmware, backend, navigator, tx_params)
+
+
+def test_sign_simple(firmware: Firmware,
+                     backend: BackendInterface,
+                     navigator: Navigator,
+                     test_name: str):
+    tx_params: dict = {
+        "nonce": NONCE2,
+        "gasPrice": Web3.to_wei(GAS_PRICE, 'gwei'),
+        "gas": GAS_LIMIT,
+        "to": ADDR2,
+        "value": 0x6f9c9e7bf61818,
```

Comment:
For all the tests in this file we could probably use `Web3.to_wei` for the `"value"`  field like in the tests above (since it makes reviewing the snapshots easier), the snapshots will probably have to be regenerated.
It does not have to use the global `AMOUNT` variable every time, it can be a different one every test, it does not really matter.

Reply:
Done